lib/timber/logger_backend.ex

```diff
@@ -179,6 +181,13 @@ defmodule Timber.LoggerBackend do
   defp output_event(ts, level, message, metadata, state) do
     %{transport: transport, transport_state: transport_state} = state
 
+    module = Keyword.get(metadata, :module)
+    fun = Keyword.get(metadata, :function)
+    file = Keyword.get(metadata, :file)
+    line = Keyword.get(metadata, :file)
+    %RuntimeContext{module: module, function: fun, file: file, line: line}
+    |> Timber.add_context()
```

Comment:
Yes, we're adding runtime information into the current context, the attributes described in the  `Console Backend` section [here](https://hexdocs.pm/logger/Logger.html). `file`, `function`, `module`, etc.

If we add this data to the current context for the line in question, we'll need to remove it afterwards to "cleanup", otherwise that context is persisted. I know how to remove it, but I wanted to make sure this wouldn't cause issue with race conditions, etc.